### DISCUSSION TITLE
Does Xournal++ work with XP pen graphic tablets & Huion Graphic tablets?

### DISCUSSION THREAD

**Discussion by TechNerdInfinity** [xournalpp/xournalpp#3485] [Q&A]
Hi,
I was selecting a graphics tablet and wanted to know whether xp-pen and huion graphic tablets work with xournal++?

**Comment by LittleHuba** [comment]
Info know that we have people using both vendors but I can't tell you how well they are supported. You can Google for GTK support though. That is the underlying library that we use for input handling.

**Reply by TechNerdInfinity** [reply]
Okay, thaks a lot

**Comment by PedroPhilipe** [comment]
I have an XP-Pen Innovator 16 and use it with Xournal++ without problems
